src/components/Search/Search.tsx

```diff
@@ -1,449 +1,67 @@
-                ))}
-              </ul>
-            </div>
-          </div>
-        }
-
-        {data.works.languages.length > 0 &&
-          <div className="panel facets add">
-            <div className="panel-body">
-              <h4>Language</h4>
-              <ul>
-                {data.works.languages.map(facet => (
-                  <li key={facet.id}>
-                    {facetLink('language', facet.id)}
-                    <div className="facet-title">{facet.title}</div>
-                    <span className="number pull-right">{facet.count.toLocaleString('en-US')}</span>
-                    <div className="clearfix"/>
-                  </li>
-                ))}
-              </ul>
-            </div>
-          </div>
-        }
-
-        <div className="panel facets add">
-          <div className="panel-body">
-            <h4>DOI Registration Agency</h4>
-            <ul>
-            {data.works.registrationAgencies.map(facet => (
-              <li key={facet.id}>
-                {facetLink('registration-agency', facet.id)}
-                <div className="facet-title">{facet.title}</div>
-                <span className="number pull-right">{facet.count.toLocaleString('en-US')}</span>
-                <div className="clearfix"/>
-              </li>
-            ))}
-          </ul>
-          </div>
-        </div>
-      </div>
-    )
-  }
   return (
-    <div className="row">
-      {renderFacets()}
-      <div className="col-md-9 panel-list" id="content">
-        <form className="form-horizontal search">
-          <input name="query" value={searchQuery || ''} onChange={onSearchChange} placeholder="Type to search..." className="form-control" type="text" />
-          <span id="search-icon" title="Search" aria-label="Search">
-            <FontAwesomeIcon icon={faSearch}/>
-          </span>
-          {searchQuery &&
-            <span id="search-clear" title="Clear" aria-label="Clear" onClick={onSearchClear}>
-              <FontAwesomeIcon icon={faTimes}/>
+    <div>
+      <Row>
+        <Col>
+          <form className="form-horizontal search">
+            <input name="query" value={searchQuery || ''} onChange={onSearchChange} placeholder="Type to search..." className="form-control" type="text" />
+            <span id="search-icon" title="Search" aria-label="Search">
+              <FontAwesomeIcon icon={faSearch} />
             </span>
+            {searchQuery &&
+              <span id="search-clear" title="Clear" aria-label="Clear" onClick={onSearchClear}>
+                <FontAwesomeIcon icon={faTimes} />
+              </span>
+            }
+          </form>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          {!searchQuery
+            ? renderIntroduction()
+            : <SearchContent searchQuery={searchQuery} />
           }
-        </form>
-
-        {renderResults()}
-      </div>
+          {/* <SearchOrganization searchQuery={searchQuery} /> */}
```

Comment:
If we are aiming to this have the similar behaviour to "Google Search", How would you toggle between the three searches ?

Reply:
Not part of this PR, the aim is to just allow seperation, we can then work on actually building a new search choosing.

Reply:
Yes, each search component handles its own querying to graphql and such, how the switch happens dont know at this point.